package.json

```diff
@@ -2,10 +2,11 @@
   "name": "emote-server",
   "version": "1.0.0",
   "description": "Server-side for Emote mobile chat application",
-  "main": "app.js",
+  "main": "server.js",
   "scripts": {
-    "start": "node app.js",
-    "test": "jshint app.js routes"
+    "start": "node src/server.js",
+    "test": "jest --verbose --runInBand --detectOpenHandles",
+    "apidoc": "apidoc -i src/routes/ -e public/ -o public/"
```

Comment:
I was previously running apidoc on a specific file with the -f option. Removed so it'll look for all apidoc-specific comments in `src/routes/`.